src/isFalse.js

```diff
@@ -0,0 +1,29 @@
+import { equals } from 'ramda';
+
+/**
+ * Checks if input value is the Boolean primative `false`. Will return false for all values created
+ * using the `Boolean` function as a constructor.
+ *
+ * @func isFalse
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.6.0|v2.6.0}
+ * @category Type
+ * @sig * -> Boolean
+ * @param {*} val The value to test
+ * @return {boolean}
+ * @see {@link https://developer.mozilla.org/en-US/docs/Glossary/Truthy|truthy}, {@link RA.isTrue|isTrue}, {@link RA.isTruthy|isTruthy}, {@link RA.isFalsy|isFalsy}
```

Comment:
Pls drop `{@link https://developer.mozilla.org/en-US/docs/Glossary/Truthy|truthy}`

It is part of `RA.isTruthy`